src/components/MovieCard.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class="movie__wrapper">
+    <h2>{{ title }}</h2>
+    <div class="image__wrapper">
+      <img :src="img" />
```

Comment:
as the `img` prop is not required it might be a good idea to hide `<img>` when it is an empty string

(psst, `v-if` is probably a good choice here)